frontend/src/data/config/DatasetMetadata.ts

```diff
@@ -172,6 +172,18 @@ export type DatasetId =
   | 'cdc_wisqars_black_men_data-black_men_by_alls_state_current'
   | 'cdc_wisqars_black_men_data-black_men_by_alls_national_historical'
   | 'cdc_wisqars_black_men_data-black_men_by_alls_state_historical'
+  | 'cdc_wonder_data-age_national_current'
+  | 'cdc_wonder_data-age_national_historical'
+  | 'cdc_wonder_data-age_state_current'
+  | 'cdc_wonder_data-age_state_historical'
+  | 'cdc_wonder_data-race_and_ethnicity_national_current'
+  | 'cdc_wonder_data-race_and_ethnicity_national_historical'
+  | 'cdc_wonder_data-race_and_ethnicity_state_current'
+  | 'cdc_wonder_data-race_and_ethnicity_state_historical'
+  | 'cdc_wonder_data-sex_national_current'
+  | 'cdc_wonder_data-sex_national_historical'
+  | 'cdc_wonder_data-sex_state_current'
+  | 'cdc_wonder_data-sex_state_historical'
```

Comment:
if we want the `alls` fallback feature to work, we need to add members here by all (and also in the DatasetMetadataMap) like `'cdc_wonder_data-alls_state_historical'`